### PR TITLE
add docstrings for legacy buildpack

### PR DESCRIPTION
**Pull request by willingc**
Partially addresses #211

### CODE REVIEW

File: repo2docker/buildpacks/legacy.py [hunk 1]
```diff
@@ -1,11 +1,17 @@
-"""
-Generates a variety of Dockerfiles based on an input matrix
+"""Generates legacy Dockerfiles based on an input matrix.
```
Comment (minrk) [inline repo2docker/buildpacks/legacy.py]: The 'input matrix' is leftover from a copy-paste when all the buildpacks were in one file. It doesn't apply here.

How about:

> Generates Dockerfiles from the legacy Binder Dockerfiles based on andrewosh/binder-base.
> The Dockerfile is amended to add the contents of the repository to the image and install a supported version of the notebook and IPython kernel.

Comment (willingc) [reply repo2docker/buildpacks/legacy.py]: Sounds much better. Thanks @minrk.

**Comment by willingc** [comment]
@minrk I'll update this again after #276 lands.

**Comment by willingc** [comment]
Updated docstrings based on changes in PR #276. Should be ready for re-review @minrk

**Pull request merged**